imap_processing/swe/swe_packet_definition.xml

```diff
@@ -0,0 +1,502 @@
+<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
Do we want each packet definition to live near the code itself, or do we want the packet definitions stored in a separate folder? I can see reasons for either one, so just curious.

`imap_processing/packet_definitions/swe_packet_definition.xml`

Reply:
Since we have moved decom code up to `imap_processing/` instead of individual folder, it might make sense to have one folder to put all packet definitions like you are suggesting. I wonder what others think.